packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -272,7 +272,7 @@ def routing_key(self) -> str | None:
 
 
 class CreditsLimit(IntEnum):
-    MIN_CREDITS = 0
+    SHUTDOWN_SERVICES = 0
```

Comment:
I am not sure I agree with that change. This messages is sent by RUT into the message broker. What you (or the dv2) do with that message is your own responsibility.

Reply:
How does this replaces MIN_CREDITS?  

This constant defined as a **minimum threshold constant** that is used as `wallet_total_credits.available_osparc_credits<CreditsLimit.MIN_CREDITS`, i.e. when the available credits are lower than the limit then do something. By default is zero but I argued in the past that this threshold should be the minimum prize of the smallest unit we have.

For starters, it is already strange that a constant like this is defined as ENUM. It works because it happens to be an int but it is confusing to interprete

What I do not understand is what SHUTDOWN_SERVICES has to do here? Is this just a rename for convenience or it has a different meaning?

Reply:
this is a value. so I think MIN_CREDITS makes more sense. this is an information event. what you do with it depends on the service. OUT_OF_CREDITS could be ok, but it also depends if there will be another message like this later.

Reply:
@GitHK so you find more intuitive reading
```python
if ( wallet_total_credits.available_osparc_credits < CreditsLimit.SHUTDOWN_SERVICES )
    ...
```
than
```python
if ( wallet_total_credits.available_osparc_credits < CreditsLimit.MIN_CREDITS )
    ...
```

I find the second more readable. But I will not push too much this, anyway.  :-)

Reply:
@pcrespov I think in you example you used my initial proposal, which I agree it's worse.

What I am asking is the below:

I think this reads better
```python
if wallet_total_credits.available_osparc_credits < CreditsLimit.OUT_OF_CREDITS:
    ...
```

than
```python
if wallet_total_credits.available_osparc_credits < CreditsLimit.MIN_CREDITS:
    ...
```

Reply:
For now it's been renamed to `OUT_OF_CREDITS`. Feel free to change it in the future if  you don't agree.